readme.md

```diff
@@ -70,18 +78,18 @@ Once installation is working, try creating and requesting resources. Note that t
 * **Question:** Describe what the above code is adding to your schema.
 
 * **Your Answer:**
-
+Defines a unique id to represent each document
```

Comment:
This is not quite right. If you include the above in your code, it will require you to send an ObjectID when creating a new record.